sdk/test-utils/recorder/src/utils.ts

```diff
@@ -300,7 +300,11 @@ export function findRecordingsFolderPath(filePath: string): string {
   try {
     // While loop to find the `recordings` folder
     while (!fs.existsSync(path.resolve(currentPath, "recordings/"))) {
-      if (fs.existsSync(path.resolve(currentPath, "package.json"))) {
+      if (
+        fs.existsSync(path.resolve(currentPath, "package.json")) &&
+        fs.existsSync(path.resolve(currentPath, "..", "..", "sdk/")) &&
```

Comment:
Just for the record, this could also be a check on package.json and node_modules. This is fine too! But, in case anyone wonders if we've talked about other approaches, we have. There are also other tools that do this that should work, but they are done by third party developers and they're arguably more complex & less straightforward.